oar/core/shipment.py

```
@@ -107,6 +108,40 @@ def get_status(self) -> str:
         except Exception as e:
             raise GitLabMergeRequestException(f"Failed to get MR status: {str(e)}")
 
+    def approve(self) -> None:
+        """Approve the current merge request (adds your approval)
+        
+        Note: 
+        - This does not merge the MR, just adds your approval
+        - The MR will still need all required approvals before it can be merged
+        - Will not approve if current user has already approved
+        
+        Raises:
+            GitLabMergeRequestException: If unable to approve the MR
+        """
+        try:
+            if self.mr.state != 'opened':
```

Can we also approve "merged"/"closed" PRs or should we raise an exception?